src/tools/emscripten.jam

```diff
@@ -94,12 +120,13 @@ actions compile.c++
 
 actions archive
 {
-    "$(CONFIG_COMMAND)" $(AROPTIONS) -r -o "$(<)" "$(>)"
+    "$(.AR)" $(AROPTIONS) rc "$(<)" "$(>)"
+    "$(.RANLIB)" "$(<)"
```

Comment:
Is ranlib really needed? AFAIK `ar s` does the same thing for a long time. See #56.

Reply:
Indeed there is such option and according to [Wikipedia](https://en.wikipedia.org/wiki/Ar_(Unix)) it seems the overall processes is geared towards `ranlib` deprecation. I'll remove it.